.gitignore

```diff
@@ -1,7 +1,10 @@
 # Development files
 .stylelintrc.json
+.stylelintrc-css.json
+.stylelintignore
 package.json
 package-lock.json
+themes/*/assets/sass
```

Comment:
I think it might be better to be more explicit here because this will impact customer themes as well:

```suggestion
themes/twentytwentyone/assets/sass
```

Reply:
I also wonder if we should make the other development files pinned to root as well? i.e. prefix with `/`

Reply:
That's a good idea, I'll add that